client/client.go

```diff
@@ -166,7 +186,7 @@ func (c *Client) SetCabinet(serial, value string) error {
 	return nil
 }
 
-// Override overrides a globally defined property such as EtcdDiscoveryURL, docker_version, yochu_version, etc
+// Override overrides a template properties such as docker_version, yochu_version, etc
```

Comment:
A host has an EtcdDiscoveryToken field now. So this is not in Overrides anymore. See https://github.com/giantswarm/mayu/pull/65/files#diff-fb55a563565044be4f4f8f0796c2013aR37